xapian-applications/omega/clickmodel/simplifieddbn.cc

```diff
@@ -148,50 +150,48 @@ SimplifiedDBN::buildSessions(string logfile)
 void
 SimplifiedDBN::train(vector<vector<string>> &sessions)
 {
-    map<string, map<string, map<char, array<double, 2>>>> urlRelFractions;
+    map<string, map<string, map<char, array<double, 2>>>> url_rel_fractions;
 
     for (size_t i = 0; i < sessions.size(); ++i) {
 	string qid = sessions[i][QID];
 
-	vector<string> docids = getDocidsList(sessions);
+	vector<string> docids = get_docid_list(sessions);
 
-	vector<int> clicks = getClicksList(sessions);
+	vector<int> clicks = get_click_list(sessions);
 
-	int lastClickedPos = clicks.size() - 1;
+	int last_clicked_pos = clicks.size() - 1;
 
 	for (size_t j = 0; j < clicks.size(); ++j)
 	    if (clicks[j] != 0)
-		lastClickedPos = j;
+		last_clicked_pos = j;
```

Comment:
You are right. It should really be `x <= last_clicked_pos` instead of `x <= last_clicked_pos + 1` here. Thanks for catching this mistake.